app/assets/javascripts/components/containers/mastodon.jsx

```diff
@@ -50,12 +50,14 @@ import es from 'react-intl/locale-data/es';
 import fa from 'react-intl/locale-data/fa';
 import fi from 'react-intl/locale-data/fi';
 import fr from 'react-intl/locale-data/fr';
+import hr from 'react-intl/locale-data/hr';
 import hu from 'react-intl/locale-data/hu';
 import it from 'react-intl/locale-data/it';
 import ja from 'react-intl/locale-data/ja';
 import pt from 'react-intl/locale-data/pt';
 import nl from 'react-intl/locale-data/nl';
 import no from 'react-intl/locale-data/no';
+import oc from 'react-intl/locale-data/oc';
```

Comment:
is this repo inclued somewhere as part of mastodon?
I mean - ok, lets say I include it and it doesn't exist, wouldn't it just fallback to smth?
if this is part of mastodon, can u locate it, I'm sure @Quent-in can contribute.